### ISSUE TITLE
Registering with assemble - 'Missing helper repeat'

### ISSUE THREAD

**Issue by tgdev** [helpers/handlebars-helper-repeat#3]
Hey @jonschlinkert,

Trying to use this helper with assemble and keep getting Missing helper 'repeat' when running grunt assemble from command line on my windows machine.

I've done npm install handlebars-helper-repeat --save-dev and it's showing in my package.json file.

i've also registered it in my grunt assemble options
helpers: ['handlebars-helper-rawinclude', 'handlebars-helper-repeat']

Finally, my handlebars template is using the repeat helper like so

{{#repeat count=3 start=1}}
...
{{/repeat}}

I've also tried your older helper `helper-repeat' which works but doesn't have the start functionality (and is 2 years old).

Could you please tell me what I'm doing wrong?

**Comment by mciastek** [comment]
I think the problem is that the helper is not wrapped in `Handlebars.registerHelper()` method. I don't why author didn't do that. Version `0.2.0` works fine, because helper logic is passed to mentioned method, but  it lacks new features.

@jonschlinkert could you make that helper work with assemblejs?

**Comment by brunowego** [comment]
+1

**Comment by jonschlinkert** [comment]
sorry, thanks for the ping. I don't know how I missed this. I'll take a look and let you know

**Comment by jonschlinkert** [comment]
Can you share more information about your setup? Assemble version, grunt-assemble, or 0.6.0? 

> I think the problem is that the helper is not wrapped in Handlebars.registerHelper() method. I don't why author didn't do that

that's a bad pattern that we used to use with grunt-assemble. If you're using grunt-assemble, the helper will need to be wrapped in a function that exports a `register` method. 

``` js
module.exports.register = function(Handlebars) {
  Handlebars.registerHelper('repeat', require('handlebars-helper-repeat'));
};
```

Again, not a great pattern, but we've fixed this in assemble 0.6.0

**Comment by jonschlinkert** [comment]
closing based on age
